pages/portfolio.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <link rel="stylesheet" href="styles/portfolio_style.css" type="text/css">
+  <link rel="icon" type="image" href="images/favicon.ico">
+  <main class="container">
+    <header class="header header-title">
+      Welcome to Kristel's Portfolio!
+    
+    </header>
+    <nav class="nav">
+      <div class="grid-container">
+        <div class="grid-item">
+          <a href="index.html">Home</a>
+        </div>
+        <div class="grid-item">
+          <a href="about.html">About</a>
+        </div>
+        <div class="grid-item">
+          <a href="portfolio.html">Portfolio</a>
+        </div>
+
+    </nav>
```

Comment:
Missing a close `div` tag.  

```suggestion
        </div>
      </div>
    </nav>
```